addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -592,6 +592,8 @@ export class HtmlField extends Component {
         // Avoid save on blur if the html field is in inline mode.
         if (!this.props.isInlineStyle) {
             this.commitChanges();
+        } else {
+            this.updateValue();
```

Comment:
Seeing this, I think the similar bug in `mass_mailing` (mentioned in the discussion on the task) is a little bit related. `mass_mailing_html_field.js` has its own `onWysiwygBlur` (passed as option to `wysiwyg`, like this file does as well), which only does `commitChanges`. It should probably get the same changes (this whole `if`/`else`) Could you give it a try and see if it solves it?

Also, nitpicking but we try to favor positive checks over negative ones.
```suggestion
        if (this.props.isInlineStyle) {
            this.updateValue();
        } else {
            this.commitChanges();
```

Reply:
The problem with that is that it reverts this commit https://github.com/odoo/odoo/commit/3f668ff3c0433302daa1283d4deff1b4c44cf59a

Reply:
How so?

Reply:
Oh, I think I misunderstood what you meant , You mean I apply the if else also the mass_mailing not the other way around , I will try it

Reply:
It didn't work , it seems that the error is coming from someplace else, with only mass_mailing installed the discard works properly without the if else ( in local) but in runbot with a lot of apps it doesn't so maybe there is another module affecting the discard of mass_mailing